website/docs/using-odo/create-component.md

```diff
@@ -3,4 +3,139 @@ title: Create Component
+```
+Alternatively, you could also use one of the existing projects on the cluster:
+```shell
+odo project list
+odo project set <project-name>  # replace <project-name> with a valid value from the list
+```
+
+odo supports interactive and non-interactive ways of creating a component.
+
+## Interactively creating a component
+
+To interactively create a component, `cd` into the cloned petclinic project (already done if you copy-pasted the command above) and execute:
+```shell
+odo create
+```
+You will be prompted with a few questions one after the another. Let's go through each one of them to create a component.
+
+1. First question is about selecting the component type:
+    ```shell
+    odo create
+    ? Which devfile component type do you wish to create  [Use arrows to move, enter to select, type to filter]
+    > java-maven
+    java-maven
+    java-openliberty
+    java-openliberty
+    java-quarkus
+    java-quarkus
+    java-springboot
+    ```
+   By default, `java-maven` is selected for us. Since this is a Spring Boot application, we should be selecting `java-springboot`. 
+
+    We can either scroll down to `java-springboot` using the arrow key, or start typing `spring` on the prompt. Typing `spring` will lead to odo filtering the component type based on your input.
+
+2. Next, odo asks you to name the component:
+    ```shell
+    odo create                
+    ? Which devfile component type do you wish to create java-springboot
+    ? What do you wish to name the new devfile component (java-springboot) petclinic
+    ```
+    Let's name it `petclinic`.
+
+3. Next, odo asks you for the project in which you would like to create the component. Use the project `myproject` that we created earlier or the one you had set using `odo project set` command
+   ```shell
+   odo create
+   ? Which devfile component type do you wish to create java-springboot
+   ? What do you wish to name the new devfile component java-springboot
+   ? What project do you want the devfile component to be created in myproject
+   ```
+   Now you will have a `devfile.yaml` in your current working directory. But odo is just not done asking you questions yet.
+4. Lastly, odo asks you if you would like to download a starter project. Since we already cloned the petclinic project, we answer in No by typing `n` and hitting the return key:
+   ```shell
+   odo create
+   ? Which devfile component type do you wish to create java-springboot
+   ? What do you wish to name the new devfile component java-springboot
+   ? What project do you want the devfile component to be created in myproject
+   Devfile Object Validation
+   ✓  Checking devfile existence [66186ns]
+   ✓  Creating a devfile component from registry: stage [92202ns]
+   Validation
+   ✓  Validating if devfile name is correct [99609ns]
+   ? Do you want to download a starter project (y/N) n
+   ```
+   
+Your Spring Boot component is now ready for use.
+
+## Non-interactively creating a component
```

Comment:
Same as above, perhaps name it **_Creating a component non-interactively_** or **_The Non-Interactive Way_** or **_Create a component: The non-interactive way_**?